agent/agent_endpoint.go

```diff
@@ -1202,6 +1203,8 @@ func (s *HTTPHandlers) AgentMonitor(resp http.ResponseWriter, req *http.Request)
 	// a gzip stream it will go ahead and write out the HTTP response header
 	resp.Write([]byte(""))
 	flusher.Flush()
+	const flushDelay = 200 * time.Millisecond
+	flushTicker := time.NewTicker(flushDelay)
```

Comment:
The docs for NewTicker say:

> Stop the ticker to release associated resources.

so I think we need to do this to prevent leaking some things on each monitor:

```suggestion
	const flushDelay = 200 * time.Millisecond
	flushTicker := time.NewTicker(flushDelay)
	defer flushTicker.Stop()
```